frontend/packages/dev-console/src/components/project-access/ProjectAccessForm.tsx

```diff
@@ -3,6 +3,7 @@ import * as _ from 'lodash';
 import { Form, TextInputTypes } from '@patternfly/react-core';
 import { FormikProps, FormikValues } from 'formik';
 import { MultiColumnField, InputField, DropdownField, FormFooter } from '@console/shared';
+import { useTranslation } from 'react-i18next';
 
 enum accessRoles {
   '' = 'Select a role',
```

Comment:
Can you please also fix this. I don't think we need this empty string value. we can pass the `title` prop to `DropdownField` instead of this.

Reply:
Hmm this is more of refactoring unrelated code for this fix but since it is a small change I have removed this and added a `title`.

Reply:
Thanks @divyanshiGupta